pilot/pkg/xds/ads.go

```diff
@@ -68,15 +68,15 @@ type DeltaDiscoveryClient = discovery.AggregatedDiscoveryService_DeltaAggregated
 
 // Connection holds information about connected client.
 type Connection struct {
-	// PeerAddr is the address of the client, from network layer.
-	PeerAddr string
+	// peerAddr is the address of the client, from network layer.
+	peerAddr string
 
 	// Time of connection, for debugging
-	Connect time.Time
+	connectedAt time.Time
 
-	// ConID is the connection identifier, used as a key in the connection table.
+	// identifier is the connection identifier, used as a key in the connection table.
 	// Currently based on the node name and a counter.
-	ConID string
```

Comment:
Suggest not changing this long-existing name

Reply:
Ok. I can revert but thought this is not used out side, it is OK to change